include/rtc/rtc.h

```diff
@@ -176,6 +199,113 @@ RTC_EXPORT int rtcDeleteTrack(int tr);
 
 RTC_EXPORT int rtcGetTrackDescription(int tr, char *buffer, int size);
 
+// Media
+#if RTC_ENABLE_MEDIA
+
+/// Add track
+/// @param pc Peer connection id
+/// @param codec Codec
+/// @param payloadType Payload type
+/// @param ssrc SSRC
+/// @param _mid MID (default: video|audio)
+/// @param _direction Direction (default RTC_SENDONLY)
+/// @param _name Name (optional)
+/// @param _msid MSID (optional)
+/// @returns Track id
+RTC_EXPORT int rtcAddTrackEx(int pc, rtcCodec codec, int payloadType, uint32_t ssrc, const char *_mid, rtcDirection * _direction, const char *_name, const char *_msid);
+
+/// Set H264PacketizationHandler for track
+/// @param tr Track id
+/// @param ssrc SSRC
+/// @param cname CName
+/// @param payloadType Payload Type
+/// @param clockRate Clock rate
+/// @param _sequenceNumber Sequence number (default: pseudo random value)
+/// @param _timestamp Timestamp (default: pseudo random value)
+RTC_EXPORT int rtcSetH264PacketizationHandler(int tr, uint32_t ssrc, const char * cname, uint8_t payloadType, uint32_t clockRate, uint16_t * _sequenceNumber, uint32_t * _timestamp);
+
+/// Set OpusPacketizationHandler for track
+/// @param tr Track id
+/// @param ssrc SSRC
+/// @param cname CName
+/// @param payloadType Payload Type
+/// @param clockRate Clock rate
+/// @param _sequenceNumber Sequence number (default: pseudo random value)
+/// @param _timestamp Timestamp (default: pseudo random value)
+RTC_EXPORT int rtcSetOpusPacketizationHandler(int tr, uint32_t ssrc, const char * cname, uint8_t payloadType, uint32_t clockRate, uint16_t * _sequenceNumber, uint32_t * _timestamp);
+
+
+/// Set onOpen callback for track with given id
+/// @param id Track identifier
+/// @param cb OnOpen callback
+int rtcSetTrackOpenCallback(int id, rtcOpenCallbackFunc cb);
+
+/// Set onClosed callback for track with given id
+/// @param id Track identifier
+/// @param cb OnClosed callback
+int rtcSetTrackClosedCallback(int id, rtcClosedCallbackFunc cb);
+
+/// Set start time for RTP stream
+/// @param startTime_s Start time in seconds
+/// @param timeIntervalSince1970 Set true if `startTime_s` is time interval since 1970, false if `startTime_s` is time interval since 1900
+/// @param _timestamp Start timestamp (default: current timestamp)
+int rtcSetRTPConfigurationStartTime(int id, double startTime_s, bool timeIntervalSince1970, const uint32_t * _timestamp);
+
+/// Start stats recording for RTCP Sender Reporter
+/// @param id Track identifier
+int rtcStartRTCPSenderReporterRecording(int id);
```

Comment:
Same, it should be written `Rtcp` instead of `RTCP` for consistency with the naming convention.